x-pack/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -467,6 +478,20 @@ function FieldItemPopoverContents(props: State & FieldItemProps) {
         </EuiText>
       </>
     );
+  } else if (field.type === 'geo_point' || field.type === 'geo_shape') {
+    return (
```

Comment:
It's possible this button is rendered even if there is no registered action (e.g. if the maps app is not available.)

Should be checked for (similar for the drop zone)

Reply:
We do not need to check if the visualize geo field action is available anywhere else because geo_point and geo_shape fields will not exist in the list when visualize geo field action is not available. Therefore, the field check will never return true since geo fields will have been removed prior to this logic.